data/pages/donate.mdx

```diff
@@ -38,17 +38,21 @@ You can donate to OpenSats via a donor-advised fund. To quote our friends from [
 There are several benefits to donating through a DAF:
 - It allows the donor to better manage their tax burden. For example, the donor may choose to donate a large sum to maximize deductions in one tax year, and then spread out their charitable giving from the fund over multiple years.
 - Any growth in the value of the fund is tax exempt.
-- It’s easy to donate [Bitcoin][daf-btc] (or other appreciated assets) to the fund while not incurring capital gains tax.
+- It’s easy to donate [bitcoin][daf-btc] (or other appreciated assets) to the fund while not incurring capital gains tax.
 
-By donating Bitcoin through your DAF, you will receive the tax-deductible
-benefit for the value of the Bitcoin, but OpenSats will receive USD, not
-Bitcoin. For more details, please contact your DAF portfolio manager.
+By donating bitcoin through your DAF, you will receive the tax-deductible
+benefit for the value of the bitcoin. Our preferred DAF partner is [Unchained
+donor-advised funds][unchained], since the bitcoin DAFs are on-chain, meaning donations to
+the DAF and grants to OpenSats can remain in bitcoin.
 
-The three largest DAFs are  [Fidelity
+With traditional DAFs, OpenSats will receive USD, not bitcoin. For more details,
+please contact your DAF portfolio manager.
+
+The three largest traditional DAFs are  [Fidelity
 Charitable](https://www.fidelitycharitable.org/),  [Schwab
 Charitable](https://www.schwabcharitable.org/)  and  [BNY Mellon Charitable Gift
```

Comment:
OpenSats uses the oxford comma in blog posts. The oxford comma was also used in the quote by Brink on this donate page. I don't think this necessarily means you need to standardize the oxford comma throughout the entire website and online content, but I am just bringing this to your attention in case you do want to do that.